packages/inventory/src/ducks/inventory.ts

```diff
@@ -55,7 +63,7 @@ export const getInventoriesByIds = (store: Partial<Store>) => {
 
 /** prop filters to customize selector queries */
 interface Filters {
-  stockIds?: string[];
+  _ids?: string[];
```

Comment:
does having this variable prefixed by an underscore potentially adds extra unwanted meaning to this variable?